tcms/telemetry/api.py

```diff
@@ -132,10 +132,19 @@ def execution_trends(query=None):
         data_set[status.name] = []
         colors.append(status.color)
 
+    status_count = {
+        'pass': 0,
+        'fail': 0
+    }
```

Comment:
Use positive/negative/neutral naming and labels.

Add the neutral statuses for completeness as well